### PR TITLE
fix for WW-4269, ww-4576, ww-4577

### PR DESCRIPTION
**Pull request by victorsosa**
blank double results in NumberFormatException being thrown on sending
JSON in request body

### CODE REVIEW

**Comment by victorsosa** [comment]
I included the ww-4576 and ww-4577

File: core/pom.xml [hunk 1]
```diff
@@ -240,6 +240,7 @@
         <dependency>
             <groupId>ognl</groupId>
             <artifactId>ognl</artifactId>
+            <version>${ognl.version}</version>
```
Comment (lukaszlenart) [inline core/pom.xml]: It isn't needed, `version` is inherited from parent pom.

Comment (victorsosa) [inline core/pom.xml]: Ok, I will remove it

Comment (victorsosa) [inline core/pom.xml]: it is ready

**Comment by victorsosa** [comment]
Glad to help, I will be committing more code

**Comment by lukaszlenart** [comment]
Thanks!

**Pull request merged**